docs/geoips_overview.rst

```diff
@@ -1,20 +1,14 @@
  | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
- | # # # 
+ | # # #
  | # # # Author:
  | # # # Naval Research Laboratory, Marine Meteorology Division
- | # # # 
- | # # # This program is free software:
- | # # # you can redistribute it and/or modify it under the terms
- | # # # of the NRLMMD License included with this program.
- | # # # 
- | # # # If you did not receive the license, see
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
  | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
- | # # # for more information.
- | # # # 
- | # # # This program is distributed WITHOUT ANY WARRANTY;
- | # # # without even the implied warranty of MERCHANTABILITY
- | # # # or FITNESS FOR A PARTICULAR PURPOSE.
- | # # # See the included license for more details.
 
 GeoIPS 2.0 Overview
```

Comment:
Quick, negligible thought-- since we've abandoned the GeoIPS2 nomenclature should existing instances of GeoIPS 2.0 be stricken?

Reply:
Agreed!